tests/testsuite/build_scripts_multiple.rs

```diff
@@ -639,27 +650,45 @@ fn bar() {
         .file("assets/foo.txt", "foo")
         .file("assets/bar.txt", "bar")
         .build();
-    p.cargo("check").run();
+    p.cargo("check")
+        .masquerade_as_nightly_cargo(&["multiple-build-scripts"])
+        .run();
 
-    // Editing foo.txt won't recompile, leading to unnoticed changes
+    // Checking again without changes won't trigger recompiling
 
-    p.change_file("assets/foo.txt", "foo updated");
     p.cargo("check -v")
+        .masquerade_as_nightly_cargo(&["multiple-build-scripts"])
         .with_stderr_data(str![[r#"
 [FRESH] foo v0.1.0 ([ROOT]/foo)
 [FINISHED] `dev` profile [unoptimized + debuginfo] target(s) in [ELAPSED]s
 
 "#]])
         .run();
```

Comment:
This no-op check run was added but we have one just before this.  How come?

Reply:
Earlier, even if we updated `foo.txt`, it didn't trigger recompiling.